### PR TITLE
New first page (fix #152)

### LINKED ISSUE #152: Make new first page with selecting source and choosing target release and arch (just one)
the second page should be the same as the first page not except loading source

### PR DESCRIPTION
**Pull request by regeciovad**

### CODE REVIEW

File: rpg/gui/wizard.py [hunk 1]
```diff
@@ -49,10 +51,135 @@ def __init__(self, Wizard, parent=None):
...
+        ''' [Bool] Function that invokes just after pressing the next button
+            {True} - user moves to next page
+            {False}- user blocked on current page
+            ###### Setting up RPG class references ###### '''
+
+        # Verifying path
+        path = Path(self.importEdit.text())
+        if(path.exists()):
+            self.base.spec.Arch = self.ArchEdit.currentText()
+            self.base.spec.Distro = self.DistroEdit.currentText()
```
Comment (jsilhan) [inline rpg/gui/wizard.py]: use `self.base.target_arch = ...` and `self.base.target_distro = ...`, please. Spec has no Arch and Distro directives/tags.

File: rpg/spec.py [hunk 2]
```diff
@@ -19,7 +19,9 @@ class Subpackage(dict):
         "Vendor",
         "Packager",
         "BuildArch",
-        "BuildRoot"
+        "BuildRoot",
+        "Arch",
```
Comment (jsilhan) [inline rpg/spec.py]: delete these, please

File: rpg/spec.py [hunk 3]
```diff
@@ -65,6 +67,8 @@ def __init__(self):
             "Vendor": "",
             "Packager": "",
             "package": "",
+            "Arch": "",
+            "Distro": "",
```
Comment (jsilhan) [inline rpg/spec.py]: delete them

**Comment by regeciovad** [comment]
Fixed

**Comment by jsilhan** [comment]
Great, can you rebase it againts upstream, please. I get conflicts. Otherwise looks good.

**Pull request merged**